Doc/library/stdtypes.rst

```diff
@@ -163,13 +163,12 @@ This table summarizes the comparison operations:
    pair: object; numeric
    pair: objects; comparing
 
-Objects of different types, except different numeric types, never compare equal.
-Furthermore, some types (for example, function objects) support only a degenerate
-notion of comparison where any two objects of that type are unequal.  The ``<``,
-``<=``, ``>`` and ``>=`` operators will raise a :exc:`TypeError` exception when
-comparing a complex number with another built-in numeric type, when the objects
-are of different types that cannot be compared, or in other cases where there is
-no defined ordering.
+Objects of different types, except different numeric types, should never compare equal.
+The ``==`` defaults to :keyword:`is` if both object's :meth:`__eq__` method returns NotImplemented.
+The ``<=``, ``>`` and ``>=`` operators aren't always defined, they will raise a
+:exc:`TypeError` exception when comparing a complex number with another built-in
+numeric type, when the objects are of different types that cannot be compared,
+or in other cases where there is no defined ordering.
```

Comment:
Should we talk about `!=` operator here? I'm not sure is `!=` always defined just like `==`.

Reply:
In new enough 3.x versions, if you define `__eq__` but leave `__ne__` undefined, `!=` falls back to `not ==`. I don't think it's worth mentioning, since this is not the language reference, just a brief summary of typical behavior of common objects.